### PR TITLE
test: engineType in getConfig response 

### PR DESCRIPTION
**Pull request by williamluke4**
Related: https://github.com/prisma/prisma-engines/pull/2175

### CODE REVIEW

File: packages/sdk/src/__tests__/engine-commands/getConfig.test.ts [hunk 1]
```diff
@@ -105,4 +105,105 @@ describe('getConfig', () => {
...
+    const libraryConfig = await getConfig({
+      datamodel: `
+      datasource db {
+        provider = "sqlite"
+        url      = "file:../hello.db"
+      }
+
+      generator gen {
+        provider = "fancy-provider"
+        engineType = "binary"
```
Comment (Jolg42) [inline packages/sdk/src/__tests__/engine-commands/getConfig.test.ts:168]:
```suggestion
        engineType = "library"
```

File: packages/sdk/src/__tests__/engine-commands/getConfig.test.ts [hunk 2]
```diff
@@ -105,4 +105,105 @@ describe('getConfig', () => {
...
+        "fromEnvVar": null,
+        "value": "file:../hello.db",
+      },
+    },
+  ],
+  "generators": Array [
+    Object {
+      "binaryTargets": Array [],
+      "config": Object {
+        "engineType": "binary",
```
Comment (Jolg42) [inline packages/sdk/src/__tests__/engine-commands/getConfig.test.ts:194]:
```suggestion
        "engineType": "library",
```

File: packages/sdk/src/__tests__/engine-commands/getConfig.test.ts [hunk 3]
```diff
@@ -105,4 +105,105 @@ describe('getConfig', () => {
 
     expect(config).toMatchSnapshot()
   })
+  test('with engineType', async () => {
```
Comment (Jolg42) [inline packages/sdk/src/__tests__/engine-commands/getConfig.test.ts:108]: Maybe split in 2 tests? one engineType=binary and the other engineType=library?

Comment (williamluke4) [reply packages/sdk/src/__tests__/engine-commands/getConfig.test.ts]: Done

File: packages/sdk/src/__tests__/engine-commands/getConfig.test.ts (hunk 2 above)
Comment (williamluke4) [reply packages/sdk/src/__tests__/engine-commands/getConfig.test.ts]: Done

File: packages/sdk/src/__tests__/engine-commands/getConfig.test.ts (hunk 1 above)
Comment (williamluke4) [reply packages/sdk/src/__tests__/engine-commands/getConfig.test.ts]: Done

**Review by Jolg42 — APPROVED** [review]

**Pull request merged**